packages/components/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,136 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconCaretDown } from '@frontify/fondue-icons';
+import * as RadixPopover from '@radix-ui/react-popover';
+import { Slot as RadixSlot } from '@radix-ui/react-slot';
+import { useSelect } from 'downshift';
+import { forwardRef, type ForwardedRef, type ReactNode } from 'react';
+
+import { ForwardedRefCombobox } from './Combobox';
+import { ForwardedRefSelectItem, ForwardedRefSelectItemGroup } from './SelectItem';
+import { SelectMenu } from './SelectMenu';
+import { ForwardedRefSelectSlot } from './SelectSlot';
+import styles from './styles/select.module.scss';
+import { useSelectData, type SelectItemType } from './useSelectData';
+
+export type SelectComponentProps = {
+    /**
+     * Children of the Select component. This can contain the `Select.Slot` components for the label, decorators, clear action and menu.
+     */
+    children?: ReactNode;
+    /**
+     * Callback function that is called when an item is selected.
+     */
+    onSelect?: (selectedItem: SelectItemType) => void;
+    /**
+     * The active item in the select component. This is used to control the select externally.
+     */
+    activeItem?: SelectItemType;
```

Comment:
`defaultValue` => `value` (`string` too) so we stay consistent?

Reply:
well, it is not a value but a complete object. you use this if you wanna control the state externally. then you need to handle the object and not only the value

Reply:
changed